### ISSUE TITLE
FS Chat no longer working

### ISSUE THREAD

**Issue by StoneColeQ** [FrankerFaceZ/Add-Ons#37]
I doesn't show up in fullscreen. Both when I have it set to FS automatically or not. Usually refreshing the page fixes it but it no longer does.

**Comment by SirStendec** [comment]
FS Chat is working fine here. Are you using any other extensions or scripts that modify the player?

**Comment by StoneColeQ** [comment]
I figured it out. It was this extension called Dark Reader which turns all webpages into dark mode even if they don't support it. I've had this extension for over 2 years so not sure why or how it even messes with FFZ now but whatever. When I tested this earlier in chrome, edge, and firefox this extension was still enabled and I missed it.

Not a big deal since both FFZ and BTTV have dark mode for twitch anyways so I just disabled it for this page. Thanks for the help.

If you want to see what happens for whatever reason here it is: https://chrome.google.com/webstore/detail/dark-reader/eimadpbcbfnmbkopoojfekhnkhdbieeh
